examples/experimental/persistent-volume-provisioning/README.md

```diff
@@ -43,34 +43,49 @@ scripts that launch kube-controller-manager.
 
 ### Admin Configuration
 
-No configuration is required by the admin!  3 cloud providers will be provided in the alpha version
-of this feature: EBS, GCE, and Cinder.
-
-When Kubernetes is running in one of those clouds, there will be an implied provisioner.
-There is no provisioner when running outside of any of those 3 cloud providers.
-
-A fourth provisioner is included for testing and development only.  It creates HostPath volumes,
-which will never work outside of a single node cluster. It is not supported in any way except for
-local for testing and development. This provisioner may be used by passing
-`--enable-hostpath-provisioner=true` to the controller manager while bringing it up.
-When using the `hack/local_up_cluster.sh` script, this flag is turned off by default.
-It may be turned on by setting the environment variable `ENABLE_HOSTPATH_PROVISIONER`
-to true prior to running the script.
-
+The admin must define `StorageClass` objects that describe named "classes" of storage offered in a cluster. Different classes might map to arbitrary levels or policies determined by the admin. When configuring a `StorageClass` object for persistent volume provisioning, the admin will need to describe the type of provisioner to use and the parameters that will be used by the provisioner when it provisions a `PersistentVolume` belonging to the class.
+
+The name of a StorageClass object is significant, and is how users can request a particular class, by specifying the name in their `PersistentVolumeClaim`. The `provisioner` field must be specified as it determines what volume plugin is used for provisioning PVs. 2 cloud providers will be provided in the beta version of this feature: EBS and GCE. The `parameters` field contains the parameters that describe volumes belonging to the storage class. Different parameters may be accepted depending on the `provisioner`. For example, the value `io1`, for the parameter `type`, and the parameter `iopsPerGB` are specific to EBS . When a parameter is omitted, some default is used.
+
+#### AWS
+
+```yaml
+kind: StorageClass
+apiVersion: extensions/v1beta1
+metadata:
+  name: slow
+provisioner: kubernetes.io/aws-ebs
+parameters:
+  type: io1
+  zone: us-east-1d
+  iopsPerGB: "10"
 ```
-env ENABLE_HOSTPATH_PROVISIONER=true hack/local-up-cluster.sh
+
+* `type`: `io1`, `gp2`, `sc1`, `st1`. See AWS docs for details. Default: `gp2`.
+* `zone`: AWS zone
```

Comment:
If zone is not specified, the provisioner picks some in the same way as 1.3. Which is quite complicated (and undocumented) process involving petsets and other magic.

https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/util.go#L212

I added `If not specified, a random zone in the same region as controller-manager will be chosen.`